packages/svelte/src/reactivity/map.js

```diff
@@ -2,15 +2,14 @@ import { DEV } from 'esm-env';
 import { source, set } from '../internal/client/reactivity/sources.js';
 import { get } from '../internal/client/runtime.js';
 import { UNINITIALIZED } from '../constants.js';
-import { map } from './utils.js';
 
 /**
  * @template K
  * @template V
  * @extends {Map<K, V>}
  */
 export class ReactiveMap extends Map {
-	/** @type {Map<K, import('#client').Source<V>>} */
+	/** @type {Map<K, import('#client').Source<symbol>>} */
 	#sources = new Map();
```

Comment:
Maybe we could use a WeakMap here to allow for object keys?

Reply:
Nah WeakMaps don't give us anything here really. It means we'd have to duplicate data as we need to store the value on the `super` and in the WeakMap, which means the WeakMap never does anything as the value is strongly held. Also WeakMaps are very slow.